python/artm/scores.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, name, class_id, topic_names, model_name, config):
             try:
                 if config.HasField('class_id'):
                     self._class_id = config.class_id
-            except:
+            except ValueError:
```

Comment:
I suggest to rewrite such try-except clause, because `HasField` method works not as someone can expect: quote from [StackOverflow](https://stackoverflow.com/a/36742793)

"I think you've misunderstood what HasField() does. It does not check whether a protobuf type defines a specific field by a name. What it does is to check if, given the name of a message field, that that field is set for the current instance."

My suggestion:


`try:`
`    self._class_id = config.class_id`
`except AttributeError:`
`    pass`